src/iFixit/Resources/config/services.yml

```diff
@@ -31,3 +31,24 @@ services:
          - ['pim_catalog_price_collection']
       tags:
         - { name: pim_catalog.attribute.comparator, priority: 1 }
+
+   # Custom Column merger that allows for nulling out prices
+   # Note: this overrides the Akeneo service by name (and extends the class)
+   pim_connector.array_converter.flat_to_standard.product.columns_merger:
+      class: iFixit\Akeneo\ColumnsMerger
+      arguments:
+         - '@pim_connector.array_converter.flat_to_standard.product.attribute_column_info_extractor'
+
+   # Decorate the price collection value factory so it doesn't complain about
+   # prices with null "amount" values.
+   ifixit_akeneo.pim.enrichment.factory.value.price_collection:
+      class: iFixit\Akeneo\PriceCollectionValueFactory
+      # Don't know why "decorates" didn't work here, Symfony says it should
+      # decorates: akeneo.pim.enrichment.factory.value.price_collection
+      arguments:
+        # This is the class we arer deocrating, so we take an instance in the
+        # constructor
+        - '@akeneo.pim.enrichment.factory.value.price_collection'
```

Comment:
IDK if I'm even looking at the right [docs](https://symfony.com/doc/current/service_container/service_decoration.html), but I thought I'd share this in case we aren't using autowiring:

> If you are not using autowiring or the decorating service has more than one constructor argument type-hinted with the decorated service class, you must inject the decorated service explicitly (the ID of the decorated service is automatically changed to '.inner'):

> ```
># config/services.yaml
>services:
 >   App\Mailer: ~
>
 >   App\DecoratingMailer:
  >      decorates: App\Mailer
  >      # pass the old service as an argument
 >       arguments: ['@.inner']
>```

Reply:
Yeah, I totally wanted to use that. I tried a bunch of times and it just never did what it claimed. It was acting like this config was loaded *before* the akeneo config, despite `Kernel.php` loading the akeneo config first explicitly.